### ISSUE TITLE
Imported 'undefined' from NativescriptDevToolsMonitors

### ISSUE THREAD

**Issue by jogboms** [vakrilov/ngrx-devtools-nativescript#3]
I have been in need of a way to track my state changes much like the ReduxDevTools until i stumbled on this tool. Its seem stale but decided to give it a go.
Then, unfortunately i got this.
```
Error: Unexpected value 'undefined' imported by the module 'NativescriptDevToolsMonitors'.
```
Currently running on the latest version of Angular and Nativescript.
Would be glad if you could help out here.

**Comment by jogboms** [comment]
Fixed this by upgrading the package to support the latest versions of its dependencies.
I created a PR that still need fixes since i still don't know how the package is supposed to work. On the console i get,
```
Dock height: 0
```

**Comment by zbarbuto** [comment]
The changes here work for me https://github.com/jogboms/ngrx-devtools-nativescript/commit/8ca9c1119e681aa48f703ec63097a4366bb7a338

@vakrilov Any chance these could be made and published?

**Comment by vakrilov** [comment]
Hi,
I'm little low on time these days, but I'll be happy to merge @jogboms PR when it's ready and republish.

**Comment by jogboms** [comment]
@vakrilov... I just opened up the pull request since @zbarbuto confirms it works for him. I still have to dig up the wrong in my own source.

**Comment by vakrilov** [comment]
Thanks for the PR.
I will try to update the examples and publish later today.

**Comment by jogboms** [comment]
All for the community.

**Comment by vakrilov** [comment]
Just published `0.4.0`
